packages/actions-global/src/hooks/use-on-action-error.ts

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import handleActionError, {
+  ActionError,
+  DispatchActionError,
+} from '../actions/handle-action-error';
+
+export default function useOnActionError(error: ActionError) {
+  const dispatch = useDispatch<DispatchActionError>();
+  return React.useCallback(() => handleActionError(error)(dispatch), [
```

Comment:
@tdeekens note that it's not `dispatch(handleActionError(error))` but `handleActionError` is a factory function